geode-core/src/test/java/org/apache/geode/distributed/internal/DistributionConfigJUnitTest.java

```diff
@@ -162,17 +159,17 @@ public void testAttributeDesc() {
 
   @Test
   public void sameCount() {
-    assertEquals(attributes.size(), setters.size());
-    assertEquals(setters.size(), getters.size());
+    assertThat(setters).hasSize(attributes.size());
+    assertThat(getters).hasSize(setters.size());
```

Comment:
There is a `hasSameSizeAs` assertion that  is cleaner and provides better failure messages:
```
assertThat(setters).hasSameSizeAs(attributes);
assertThat(getters).hasSameSizeAs(setters);
```